Dockerfile

```diff
@@ -15,6 +17,7 @@ RUN \
   DEBIAN_FRONTEND=noninteractive \
     apt-get -y install \
       apache2 \
+      mysql-client \
```

Comment:
Tilføj gerne lige en kommentar om hvorfor apache-imaget skal have den her (.. "drush har brug for den for at virke" er vel ca ok)